libs/partners/box/pyproject.toml

```diff
@@ -0,0 +1,97 @@
+[tool.poetry]
+name = "langchain-box"
+version = "0.1.0"
+description = "An integration package connecting Box and LangChain"
+authors = []
+readme = "README.md"
+repository = "https://github.com/langchain-ai/langchain"
+license = "MIT"
+
+[tool.poetry.urls]
+"Source Code" = "https://github.com/langchain-ai/langchain/tree/master/libs/partners/box"
+
+[tool.poetry.dependencies]
+python = ">=3.9.0,<3.13"
+langchain-core = "^0.2.0"
+langchain-community = "^0.2.0"
+box-sdk-gen = {extras = ["jwt"], version = "^1.1.0"}
+unstructured = {extras = ["all-docs"], version = "^0.15.1"}
```

Comment:
should we use the `langchain-unstructured` integration instead? Not sure it makes sense to have this as a hard dependency of the box integration. Esp with `all-docs`, this is a very heavy dependency.